canonical-websites/cn.ubuntu.com
# add new Orange Pi snap store press release

## Done

Added a new news story/press release

## QA

Check `/news/2017/03/16/orange-pi-snap-store` against the [copydoc](https://docs.google.com/document/d/1kFSDEfSIeDbE5DPi62qlFtv0zlklPOPc9ewTwk_kIiw/edit?ts=58c7f7db)

[Demo](http://cn.ubuntu.com-orange-pi-snap-store.demo.haus/news/2017/03/16/orange-pi-snap-store/)
